web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -44,7 +44,8 @@ import OSKLayer from './keyboard-layout/oskLayer.js';
 import OSKLayerGroup from './keyboard-layout/oskLayerGroup.js';
 import { LengthStyle, ParsedLengthStyle } from './lengthStyle.js';
 import { defaultFontSize, getFontSizeStyle } from './fontSizeUtils.js';
-import InternalKeyTip from './input/gestures/browser/keytip.js';
+import PhoneKeyTip from './input/gestures/browser/keytip.js';
+import { TabletKeyTip } from './input/gestures/browser/tabletPreview.js';
```

Comment:
Stupid beginner question: why is one in braces, the other not?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import

The `default export` from a ES module does not need to be wrapped in curly braces.  It can then be renamed without any obvious aliasing.

General style guidelines and recommendations these days generally eschew use of `default` imports and exports, but I hadn't run into that in some of the earlier modularization work.  Perhaps I should do a round of dropping the `default`s I set up earlier on before we release 17.0, but that's a separate issue from this PR.  Newer stuff in our codebase should be written with explicitly-named exports, rather than using the `export default ...` syntax.